Quantum-Control-Applications-QuAM/Superconducting/quam_builder/architecture/superconducting/qpu/base_quam.py

```diff
@@ -64,36 +58,29 @@ def load(cls, *args, **kwargs) -> "QuAM":
             if quam_state_path is None:
                 raise ValueError(
                     "No path argument provided to load the QuAM state. "
-                    "Please provide a path or set the 'QUAM_STATE_PATH' environment variable. "
+                    "Please set the quam_state_path in the QUAlibrate config. "
                     "See the README for instructions."
                 )
 
             args = (quam_state_path,)
 
         return super().load(*args, **kwargs)
 
-    @classmethod
-    def load(cls, *args, **kwargs) -> "QuAM":
-        if not args:
-            if "QUAM_STATE_PATH" in os.environ:
-                args = (os.environ["QUAM_STATE_PATH"],)
-            else:
-                raise ValueError(
-                    "No path argument provided to load the QuAM state. "
-                    "Please provide a path or set the 'QUAM_STATE_PATH' environment variable. "
-                    "See the README for instructions."
-                )
-        return super().load(*args, **kwargs)
-
```

Comment:
what's happening to the load method? Don't we still need it because otherwise it doesn't know where to get the quam state path from?